tools/harness-thci/OpenThread.py

```diff
@@ -2963,13 +2963,19 @@ def setBbrDataset(self, SeqNumInc=False, SeqNum=None, MlrTimeout=None, ReRegDela
             False: fail to set BBR Dataset
         """
         assert not (SeqNumInc and SeqNum is not None), "Must not specify both SeqNumInc and SeqNum"
+
+        if MlrTimeout and MlrTimeout != self.bbrMlrTimeout or ReRegDelay and ReRegDelay != self.bbrReRegDelay:
```

Comment:
I wonder if we should set SeqNumInc=True if SeqNum is already specified?

Reply:
This is not the case in any of the existing scripts, but the check can be added.